src/psyclone/tests/psyir/frontend/fparser2_program_handler_test.py

```diff
@@ -47,6 +47,20 @@
 from psyclone.psyir.symbols import RoutineSymbol
 
 
+def test_empty_program_handler(parser):
+    '''Test that an empty program handler still returns a FileContainer
```

Comment:
"the program handler still returns a FileContainer for an empty program" perhaps?

Reply:
Yes, that's better